packages/upset/src/components/Sidebar.tsx

```diff
@@ -33,12 +33,19 @@ import { hideEmptySelector, maxVisibleSelector, minVisibleSelector } from '../at
 import { sortBySelector } from '../atoms/config/sortByAtom';
 import { visibleSetSelector } from '../atoms/config/visibleSetsAtoms';
 import { ProvenanceContext } from './Root';
-import { exportStateGrammar, ImportModal } from './ImportModal';
+import { ImportModal } from './ImportModal';
 import { provenanceVisAtom } from '../atoms/provenanceVisAtom';
 import { elementSidebarAtom } from '../atoms/elementSidebarAtom';
+import { contextMenuAtom } from '../atoms/contextMenuAtom';
+import { rowsSelector } from '../atoms/renderRowsAtom';
+import { exportState } from '../utils/exports';
+
+type Props = {
+  data: CoreUpsetData
+}
 
 /** @jsxImportSource @emotion/react */
-export const Sidebar = () => {
+export const Sidebar: FC<Props> = ({ data }) => {
```

Comment:
Using the data as a prop has consequences in case we ever need to refactor/move this component. Would it better to get the data from recoil? Can we do that or is it just rows that are stored there?

Reply:
From what I can find, I'm not able to access the dataAtom.ts in the Upset package. I could make an atom that stores it in the Upset package if that is a better method than passing data to Sidebar